frontend/src/app/supportAdmin/DeviceType/ManageDevicesForm.tsx

```diff
@@ -160,6 +162,28 @@ const ManageDevicesForm: React.FC<Props> = ({
                   />
                 </div>
               </div>
+              <div className="grid-row grid-gap">
+                <div
+                  className="tablet:grid-col"
+                  style={{ marginBottom: "56px" }}
+                >
+                  <MultiSelect
```

Comment:
There doesn't seem to be any frontend form validation against the required fields on this managed devices screen. I know we're assuming null values in the supported diseases to field to default to COVID until can update it manually, but unsure if this is intentional for the other fields. If it's not intentional, might be something we want to add to the other fields just to prevent human error (especially since I'll probably be the one interacting against this form 😅)

<img width="952" alt="image" src="https://user-images.githubusercontent.com/29645040/161130036-4bf86a50-fa14-4d8a-9082-a3c41646368d.png">

Reply:
You're right - we don't have any frontend validation here. I believe we just haven't prioritized adding frontend validation for the forms in the Admin pages because the users are fewer and more tech savvy. I think it makes sense to add, but I don't want it to slow down the multiplex work - would you mind writing up a ticket for that? Good callout!